pcsx2/Achievements.cpp

```diff
@@ -2301,6 +2303,12 @@ void Achievements::RAIntegration::RACallbackWriteMemory(unsigned int address, un
 	PokeMemory(address, sizeof(value), nullptr, static_cast<unsigned>(value));
 }
 
+s32 Achievements::GetNotificationsDuration()
+{
+	//Clamp in case setting was updated manuallying using the INI
+	return std::clamp(EmuConfig.Achievements.NotificationsDuration, 3, 10);
```

Comment:
Probably better to clamp this at load time (in `LoadSave()`), gate it by `IsLoading()`.

Reply:
That's a great point. I think I found an example of how to get that done, so I'll take a stab at it.